pkg/aggregator/no_aggregation_stream_worker.go

```diff
@@ -195,14 +195,18 @@ func (w *noAggregationStreamWorker) run() {
 							sample.GetTags(w.taggerBuffer, w.metricBuffer)
 							w.metricBuffer.AppendHashlessAccumulator(w.taggerBuffer)
 
+							// if the value is a rate, we have to account for the 10s interval
+							if mtype == metrics.APIRateType {
+							    sample.Value /= 10
```

Comment:
Perhaps?
```suggestion
							    sample.Value /= bucketSize
```

Reply:
Good idea, let me check if `serie.Interval = bucketSize` is fully correct as well then.